{
  "contest": 420,
  "handle": "mashbust",
  "risk": "3",
  "title": "Potential Denial of Service (DoS) by Forger in the forgeWithListed()",
  "issueId": 1257,
  "issueUrl": "https://github.com/code-423n4/2024-07-traitforge-validation/issues/1257"
}